eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -66,14 +66,9 @@ stages:
                         PackageName: ${{artifact.name}}
                         ServiceName: ${{parameters.ServiceDirectory}}
                         ForRelease: true
-                    - template: /eng/pipelines/templates/steps/stage-artifacts.yml
```

Comment:
can we delete the stage-artifacts.yml now?

Reply:
yes